WalletWasabi.Fluent/Extensions/AvaloniaExtensions.cs

```diff
@@ -15,4 +16,30 @@ public static class AvaloniaExtensions
 			add => target.AddHandler(routedEvent, add, routingStrategies),
 			remove => target.RemoveHandler(routedEvent, remove));
 	}
+
+	public static void BringToFront(this Window? window)
+	{
+		if (window is null)
+		{
+			return;
+		}
+
+		if (!window.IsActive)
+		{
+			var position = window.Position;
+			if (window.IsVisible)
+			{
+				window.Hide();
+			}
+
+			window.Position = position;
+			window.Show();
+		}
+
+		window.Topmost = true;
+		if (window.Topmost)
```

Comment:
Just a question, why do we set `window.Topmost` to `true` then we check if it is true or not?

Reply:
To silence DS

Reply:
DS is stupid in this case. And also it won't work if `Topmost = true` by default. This is not the case now, but we don't what will be in the future.